lighthouse-core/test/audits/third-party-summary-test.js

```diff
@@ -37,18 +37,46 @@ describe('Third party summary', () => {
         mainThreadTime: 127.15300000000003,
         blockingTime: 18.186999999999998,
         transferSize: 30827,
+        subItems: {
+          items: [
+            {
+              blockingTime: 18.186999999999998,
+              mainThreadTime: 127.15300000000003,
+              transferSize: 30827,
+              url: 'https://www.googletagmanager.com/gtm.js?id=GTM-Q5SW',
+            },
+          ],
+          type: 'subitems',
+        },
       },
       {
         entity: {
           text: 'Google Analytics',
           type: 'link',
           url: 'https://www.google.com/analytics/analytics/',
         },
-        mainThreadTime: 95.15600000000005,
+        mainThreadTime: 95.15599999999999,
         blockingTime: 0,
         transferSize: 20913,
+        subItems: {
+          items: [
+            {
+              blockingTime: 0,
+              mainThreadTime: 55.246999999999986,
+              transferSize: 12906,
+              url: 'https://www.google-analytics.com/analytics.js',
+            },
+            {
+              blockingTime: 0,
+              transferSize: 8007,
+              url: expect.any(String),
+            },
+          ],
+          type: 'subitems',
+        },
       },
     ]);
+    expect(results.details.items[1].subItems.items[1].url).toBeDisplayString('Other resources');
```

Comment:
heh, I bet our users just *love* the ergonomics of working with subitems 😆

Reply:
oof yeah